tests/integration/targets/dns_delegation/tasks/main.yml

```diff
@@ -0,0 +1,277 @@
+          - delegation_info.objects[0].delegation_servers[0].fqdn == "ns1.com."
+          - delegation_info.objects[0].comment == "Delegation zone is created by Ansible"
+
+    - name: Create DNS Delegation with Disabled set to false
+      infoblox.bloxone.dns_delegation:
+        fqdn: "{{ delegation_fqdn }}"
+        comment: "Delegation zone is created by Ansible"
+        disabled: false
+        delegation_servers:
+          - address: "12.0.0.0"
+            fqdn: "ns1.com."
+        state: present
+      register: delegation
+    - name: Get Information about the DNS Delegation
+      infoblox.bloxone.dns_delegation_info:
+        filters:
+          fqdn: "{{ delegation_fqdn }}"
+      register: delegation_info
+    - assert:
+        that:
+          - delegation_info is not failed
+          - delegation_info.objects | length == 1
+          - delegation_info.objects[0].id == delegation.id
+          - delegation_info.objects[0].disabled == false
+          - delegation_info.objects[0].comment == "Delegation zone is created by Ansible"
+
+    - name: Create DNS Delegation with FQDN
+      infoblox.bloxone.dns_delegation:
+        fqdn: "{{ delegation_fqdn }}"
+        comment: "Delegation zone created with Ansible for FQDN test"
+        delegation_servers:
+          - address: "12.0.0.0"
+            fqdn: "ns1.com."
+        state: present
+      register: delegation
+    - name: Get Information about the DNS Delegation
+      infoblox.bloxone.dns_delegation_info:
+        filters:
+          fqdn: "{{ delegation_fqdn }}"
+      register: delegation_info
+    - assert:
+        that:
+          - delegation_info is not failed
+          - delegation_info.objects | length == 1
+          - delegation_info.objects[0].id == delegation.id
+          - delegation_info.objects[0].fqdn == delegation_fqdn
+          - delegation_info.objects[0].comment == "Delegation zone created with Ansible for FQDN test"
+
+    - name: Create DNS Delegation with Tags
+      infoblox.bloxone.dns_delegation:
+        fqdn: "{{ delegation_fqdn }}"
+        comment: "Delegation zone created with Ansible for tags test"
+        delegation_servers:
+          - address: "12.0.0.0"
+            fqdn: "ns1.com."
+        tags:
+          location: "site-1"
+        state: present
+      register: delegation
+    - name: Get Information about the DNS Delegation
+      infoblox.bloxone.dns_delegation_info:
+        filters:
+          fqdn: "{{ delegation_fqdn }}"
+      register: delegation_info
+    - assert:
+        that:
+          - delegation_info is not failed
+          - delegation_info.objects | length == 1
+          - delegation_info.objects[0].id == delegation.id
+          - delegation_info.objects[0].fqdn == delegation_fqdn
+          - delegation_info.objects[0].comment == "Delegation zone created with Ansible for tags test"
+          - delegation_info.objects[0].tags.location == "site-1"
+
+    - name: Create DNS Delegation in view
```

Comment:
Since the view isnt being changed here , it might just be a good idea to add the check in any other UT

Reply:
done, removed UT and added the check in the beginning